src/css/mapbox-gl.css

```diff
@@ -769,3 +769,27 @@ a.mapboxgl-ctrl-logo.mapboxgl-compact {
         display: none;
     }
 }
+
+.mapboxgl-scroll-zoom-blocker-control {
+    color: #fff;
+    font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Helvetica, Arial, sans-serif;
+    font-size: 100%;
+    justify-content: center;
+    position: absolute;
+    display: flex;
+    align-items: center;
+    top: 0;
+    left: 0;
+    width: 100%;
+    height: 100%;
+    background: rgba(0, 0, 0, 0.5);
+    opacity: 0;
+    z-index: 3;
```

Comment:
I think we can remove the z-index since this element is after the canvas. See: [Absolutely-positioned elements with the same z-index: which element will be on top?](https://stackoverflow.com/questions/12695313/absolutely-positioned-elements-with-the-same-z-index-which-element-will-be-on-t)
```suggestion
```